app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::ReviewsController < ApiController
+  protect_from_forgery unless: -> {request.format.json? }
+
+  def show 
+    render json: review.all
+  end
+
+  def create
+    review = Review.new(review_params)
+    review.sauce_id = params[:sauce_id]
+    if review.save
+      render json: {review: review}
+    else 
+      render json: {error: review.errors.full_messages }, status: :unprocessable
+    end 
+  end
```

Comment:
a review should be linked to a user (specifically the current_user!)

Reply:
Nick, we have not linked users to reviews yet. Would you like this to be done prior to submitted or is that a separate feature?

Reply:
added user associations